### ISSUE TITLE
[Suggestion]: Database Tables Unique Names Suggestions.. 

### ISSUE THREAD

**Issue by DavidChriss** [Lakshan-Madushanka/laravel-comments#55]
### Bug type

Behaviour

### Commenter Version

2.0.6

### Database

mysql

### Description

Hello I have some suggestion for Laravel Commenter... First of all Thanks for the nice comment system for Laravel and Nice support

However I have a suggestion for database table names... instead of using general table names like "**reactions**" "**comments**" etc.. 

You should use unique name to avoid conflict with user database table names for example

**Like / Example** : "**commenter_reactions**", "**commenter_comments**", "commenter_guests" etc

In my case I was having "**reactions**" table already so to do migration I decided to rename my theme reactions table to **post_reactions** just to be able to use commenter.. and between those two they were having different structure so I can't let them share the same database name.. That require me to rename one of them

Thank you ❤️

**Comment by Lakshan-Madushanka** [comment]
You can do that via config https://github.com/Lakshan-Madushanka/laravel-comments/blob/456e007dff37347b817563af462452a830be5563/config/comments.php#L52

**Comment by Lakshan-Madushanka** [comment]
Updated documentation with detailed instructions https://lakm.gitbook.io/commenter/configuration/migrations-tables.

**Comment by Lakshan-Madushanka** [comment]
This was intentionally avoided during package's design phrase.

If we mixed both mode, It would lead to security vulnarabilities. We use auth mode for specially for the users who have access to our system. If we allowed guests alongside, It would violate the intergrity of the system.

In your example, a guest can impose your email address and make some wicked commets on your behalf to damage your reputation.
